### PR TITLE
user-info microservice #60

### PR DESCRIPTION
**Pull request by surajp28**

### CODE REVIEW

File: user-service/api/routes/user.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+router.get('/', (req, res, next) =>{                    //To get all users
+    User.find()
+    .exec()
+    .then(docs => {
+      console.log(docs);
+      //   if (docs.length >= 0) {
+      res.status(200).json(docs);
+      //   } else {
+      //       res.status(404).json({
```
Comment (visheshdembla) [inline user-service/api/routes/user.js:15]: could you remove all the unnecessary comments

File: user-service/api/routes/user.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+
+module.exports = router;
+
+/*
+1. You do not return anything from those methods since returning more than one response is not a good idea.
+2. Also since there is just one item inside the json method it directly returns that
+3. Npm is a dependency tool from where you download all the dependencies.
+4. The npm commanded created a json folder. You should do it too
+5. We are using express framework for nodeJs
```
Comment (visheshdembla) [inline user-service/api/routes/user.js:120]: and these as well

File: user-service/app.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    res.json({
+        error: {
+            message: error.message
+        }
+    });
+});
+
+module.exports = app;
+
+//nodemon was used so that we dont need to restart the service by stopping it again and starting
```
Comment (visheshdembla) [inline user-service/app.js:48]: here as well

**Review by visheshdembla — CHANGES_REQUESTED** [review]

**Comment by visheshdembla** [comment]
@surajp28 are the changes for the service done or should I take care of these while converting the service to gRPC?

**Comment by surajp28** [comment]
@surajp28 Sorry I couldnt finish these changes. Could you handle them while converting the service to gPRC?

**Comment by visheshdembla** [comment]
will make the changes as part of the next issue created to convert the service to a gRPC based one

**Review by visheshdembla — APPROVED** [review]

**Pull request merged**